modules/openapi-generator/src/test/resources/bugs/issue_13241.yaml

```diff
@@ -0,0 +1,17 @@
+openapi: "3.0.0"
+info:
+  version: 2.0.0
+  title: test
+paths:
+  /pets:
+    get:
+      summary: List all pets
+      operationId: listPets
+      responses:
+        '200':
+          description: OK
+components:
+  schemas:
+    Color:
```

Comment:
Can you add another enum that is nullable so we can test that as well?

Would also be good to validate that `required: true` doesn't allow UNKNOWN since the field would be required.

Reply:
Testing a nullable enum returns null. This relates to the conversation above and I will make the change to return the `defaultCase` if needed.

The `required: true` is a tricky one as it kind of contradicts itself. You really shouldn't have both options configured if you must select an option from the enum. In the event that you do, I would suggest that the `defaultCase `returning is correct. As why set it otherwise, what do you think?

Reply:
This is the behavior with `required ` and `enumUnknownDefaultCase` set to `true`. 

```
    public enum ColorsEnum {

        RED("RED"), BLUE("BLUE"), GREEN("GREEN"), UNKNOWN_DEFAULT_OPEN_API("unknown_default_open_api");
```
and

```
    /**
     * Get colors
     * @return colors
     */
    @NotNull
    @Schema(name = "colors", required = true)
    public JsonNullable<ColorsEnum> getColors() {
        return colors;
    }
```